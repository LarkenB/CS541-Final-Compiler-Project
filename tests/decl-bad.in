int y = x;
float y = 0.1;