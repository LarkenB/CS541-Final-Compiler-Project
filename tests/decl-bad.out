target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
target triple = "x86_64-pc-linux-gnu"

define dso_local i32 @main() {
  %t1 = alloca float
  store float 0x3FB99999A0000000, float* %t1
  ret i32 0
}
